### PR TITLE
Update Ballerina Integrator VSCode README files

### LINKED ISSUE #300: Update the Ballerina Integrator VSCode extension's home page
**Description:**
* Update the content of the extension's home page.
* Specify the Ballerina version.
* "Go to VSCode Command Pallete” in the extension's home page should include the shortcut keys as well.
* Name of the VSCode Extension should be Ballerina Integrator.
* Default README can have the installation steps, but these need to be removed in the one that is bundled.
* Include gifs demonstrating on how the extension is to be used.
* Command shortcuts of Ballerina can be included in the Ballerina Integrator.

**Affected Product Version:**
Ballerina Integrator VSCode extension

**Related Issues:**
https://github.com/wso2/ballerina-integrator/issues/39

### PR DESCRIPTION
**Pull request by IrushiL**
## Purpose
* Add a separate README which is to be bundled through the VSCode Extension, apart from the one for the repository.
* Make improvements and modifications representing changes in the VSCode extension.

resolves #300

### CODE REVIEW

File: vscode/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+### Installation
+* **Using the VSCode Marketplace**
+
+    Install the `Ballerina Integrator` VSCode Extension by WSO2.
+*Please note that this extension has not been released to the VSCode Marketplace yet.*
+
+* **Using a .vsix File**
+   * Prerequisites:
+     - Node.js / NPM
+     - Yeoman and  VSCode Extension Generator
```
Comment (samgnaniah) [inline vscode/README.md]:
```suggestion
     - Yeoman and VS Code Extension Generator
```

File: vscode/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        ```
+        npm install -g yo generator-code
+        ```
+     - Visual Studio Code Extension Manager
+        ```
+        npm install -g vsce
+        ```
+   - Clone [this](https://github.com/wso2/ballerina-integrator) repository and open the `vscode/extension` folder in Visual Studio Code.
```
Comment (samgnaniah) [inline vscode/README.md]:
```suggestion
   - Clone [the Ballerina Integrator repository](https://github.com/wso2/ballerina-integrator) and open the `vscode/extension` directory in Visual Studio Code.
```

File: vscode/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+   - Load the required node modules using the following command.
+        ```
+        npm install
+        ```
+   - Generate a .vsix file using the following command.
+        ```
+        vsce package
+        ```
+   - Install the extension using `Extensions: Install from VSIX...` command in the VSCode Command Pallete and by selecting the .vsix file created in the previous step.
```
Comment (samgnaniah) [inline vscode/README.md]: Can we add a sample command for this?

File: vscode/README.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+- **Step 1:** Install the WSO2 Ballerina Integrator extension for Visual Studio Code.
+- **Step 2:** Use the command `Ballerina Integrator: Home` through the command palette to start creating projects and modules using multiple different templates provided by WSO2.
+
+### Features
+- Create New Project
+- Create New Module from Template
+- Search for Templates
+
+## Contributing Templates
+Create a Ballerina project with a template module for an integration scenarios.
```
Comment (samgnaniah) [inline vscode/README.md]:
```suggestion
Create a Ballerina project with a template module for an integration scenario.
```

File: vscode/README.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+Add the following property in the project's Ballerina.toml file.
+```
+templates= ["template_module_name"]
+```
+Build the template using the following command.
+```
+ballerina build <module_name>
+```
+Push the template module to Ballerina Central through a registered organization using the following command at the project's base folder. More details can be found [here](https://v1-0.ballerina.io/learn/).
```
Comment (samgnaniah) [inline vscode/README.md]: Let's point to https://v1-0.ballerina.io/learn/how-to-publish-modules/

File: vscode/extension/README.md [hunk 6]
```diff
@@ -1,86 +1,33 @@
-# Ballerina-Integrator VSCode Extension
-### 1.0.0
-Ballerina Integrator VSCode Extension is a tool that could be used alongside the Ballerina programming language to ease the creation of integration projects. This extension provides templates for integration use cases which could be used to create and start off new projects. It also contains code snippets for commonly used integration scenarios.
+# WSO2 Ballerina Integrator
+## Description
+WSO2 Ballerina Integrator VSCode extension is a tool that could be used alongside the Ballerina programming language to ease the creation of integration projects. This extension provides easy access to WSO2 module templates in Ballerina Central for integration use cases, which could be used to create and start new projects. Templates that demonstrate various use cases of Ballerina connectors provided by WSO2 can also be found here.
```
Comment (samgnaniah) [inline vscode/extension/README.md]:
```suggestion
WSO2 Ballerina Integrator VS Code extension is a tool that is used alongside the Ballerina programming language to simplify the creation of integration projects. This extension provides easy access to WSO2 module templates in Ballerina Central for integration use cases, which could be used to create and start new projects. You can also find templates that demonstrate various use cases of Ballerina connectors provided by WSO2.
```

File: vscode/README.md (hunk 1 above)
Comment (IrushiL) [reply vscode/README.md]: Done

File: vscode/README.md (hunk 2 above)
Comment (IrushiL) [reply vscode/README.md]: Done

File: vscode/README.md (hunk 3 above)
Comment (IrushiL) [reply vscode/README.md]: "Extensions: Install from VSIX..." is the command

File: vscode/README.md (hunk 4 above)
Comment (IrushiL) [reply vscode/README.md]: Done

File: vscode/README.md (hunk 5 above)
Comment (IrushiL) [reply vscode/README.md]: Done

File: vscode/extension/README.md (hunk 6 above)
Comment (IrushiL) [reply vscode/extension/README.md]: Done

File: vscode/README.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
+# WSO2 Ballerina Integrator
+## 1.0.0
+
+Ballerina Integrator VSCode Extension is a tool that could be used alongside the Ballerina programming language to ease the creation of integration projects. This extension provides templates for integration use cases that could be used to create and start new projects. It also contains code snippets for commonly used integration scenarios.
```
Comment (samgnaniah) [inline vscode/README.md]:
```suggestion
Ballerina Integrator VS Code Extension is a tool that can be used alongside the Ballerina programming language to simplify the creation of integration projects. This extension provides templates for integration use cases that could be used to create and start new projects. It also contains code snippets for commonly used integration scenarios.
```

File: vscode/README.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+### Requirements
+  - Visual Studio Code
+  - Ballerina (v1.0.0)
+  - Ballerina VSCode Extension (v1.0.0)
+
+### Installation
+* **Using the VSCode Marketplace**
+
+    Install the `Ballerina Integrator` VSCode Extension by WSO2.
```
Comment (samgnaniah) [inline vscode/README.md]:
```suggestion
    Install the `Ballerina Integrator` VS Code Extension by WSO2.
```

File: vscode/README.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+## 1.0.0
+
+Ballerina Integrator VSCode Extension is a tool that could be used alongside the Ballerina programming language to ease the creation of integration projects. This extension provides templates for integration use cases that could be used to create and start new projects. It also contains code snippets for commonly used integration scenarios.
+
+## User Guide
+
+### Requirements
+  - Visual Studio Code
+  - Ballerina (v1.0.0)
+  - Ballerina VSCode Extension (v1.0.0)
```
Comment (samgnaniah) [inline vscode/README.md]:
```suggestion
  - Ballerina VS Code Extension (v1.0.0)
```

File: vscode/README.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+### Installation
+* **Using the VSCode Marketplace**
+
+    Install the `Ballerina Integrator` VSCode Extension by WSO2.
+*Please note that this extension has not been released to the VSCode Marketplace yet.*
```
Comment (samgnaniah) [inline vscode/README.md]:
```suggestion
*Please note that this extension has not been released to the VS Code Marketplace yet.*
```
Can we fix everywhere? It is normally written as "VS Code".

File: vscode/README.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+* **Using a .vsix File**
+   * Prerequisites:
+     - Node.js / NPM
+     - Yeoman and  VS Code Extension Generator
```
Comment (samgnaniah) [inline vscode/README.md]:
```suggestion
     - Yeoman and VS Code Extension Generator
```

File: vscode/README.md (hunk 7 above)
Comment (IrushiL) [reply vscode/README.md]: Done

File: vscode/README.md (hunk 9 above)
Comment (IrushiL) [reply vscode/README.md]: Done

File: vscode/README.md (hunk 8 above)
Comment (IrushiL) [reply vscode/README.md]: Done

File: vscode/README.md (hunk 10 above)
Comment (IrushiL) [reply vscode/README.md]: Done

File: vscode/README.md (hunk 11 above)
Comment (IrushiL) [reply vscode/README.md]: Done

**Pull request merged**